mlbench/refimpls/pytorch/utils/utils.py

```diff
@@ -0,0 +1,8 @@
+def convert_dtype(options, obj):
```

Comment:
I think it would be cleaner to have it `def convert_dtype(dtype, obj):` and call it as `convert_dtype(options.dtype)`, then it's more clear what it's doing and not dependent on implementation details of `options`